### PR TITLE
Correct triangle facts

### PR DESCRIPTION
**Pull request by mostlybadfly**
Sorry for all the commits :) I was having some weird problem with my indentations changes actually recording.  I think I got it.

### CODE REVIEW

File: triangle_facts/triangle_facts.rb [hunk 1]
```diff
@@ -1,58 +1,61 @@
...
+
+  def equilateral?
+    side1 == side2 && side2 == side3
+  end
+
+  def isosceles?
+    [side1, side2, side3].uniq.length == 2
+  end
+
+  def scalene?
```
Comment (paulghaddad) [inline triangle_facts/triangle_facts.rb]: I believe this should be `!(equilateral? || isosceles?)`

File: triangle_facts/triangle_facts.rb [hunk 2]
```diff
@@ -1,58 +1,61 @@
...
+  def recite_facts
+    puts "This triangle is equilateral!" if equilateral?
+    puts "This triangle is isosceles! Also, that word is hard to type." if isosceles?
+    puts "This triangle is scalene and mathematically boring." if scalene?
+
+    angles = calculate_angles(side1, side2, side3)
+
+    puts "The angles of this triangle are #{angles.join(', ')}" 
+    puts "This triangle is also a right triangle!" if right_triangle?
+    puts ''
```
Comment (paulghaddad) [inline triangle_facts/triangle_facts.rb]: A simple `puts` would work here.

File: triangle_facts/triangle_facts.rb [hunk 3]
```diff
@@ -1,58 +1,61 @@
...
+  end
+
+  def radians_to_degrees(rads)
+    (rads * 180 / Math::PI).round
+  end
+
+  private
```
Comment (paulghaddad) [inline triangle_facts/triangle_facts.rb]: Line break needed after this.

**Comment by paulghaddad** [comment]
@mostlybadfly Overall, you got this right on the mark. The only major issue is your `scalene` method returning the opposite of what it should. You're good to move on!

**Comment by mostlybadfly** [comment]
Great, thanks @paulghaddad !

**Comment by paulghaddad** [comment]
@mostlybadfly Hey Dan...do you need any help in moving along with Level Up?

**Comment by mostlybadfly** [comment]
@paulghaddad Hey thanks for checking in, just have been very busy with other projects, but hope to start up the Robot Name exercise soon. I'll reach out for help on it.

**Pull request closed without merging**